src/strings/en-us.json

```diff
@@ -40,6 +40,7 @@
     "AspectRatio": "Aspect ratio",
     "AttributeNew": "New",
     "Audio": "Audio",
+    "AuthProviderHelp": "Select an Authentication Provider to be used to Authenticate this user's password",
```

Comment:
Capitalization of `Authenticate` should probably be removed.

Reply:
```suggestion
     "AuthProviderHelp": "Select an Authentication Provider to be used to authenticate this user's password", 
```